benchmark/kvbench/commands/nixlbench.py

```
@@ -228,13 +228,17 @@ def _configure_ucx(self, backend: str, source: str, destination: str):
+        is implicit from the backend, not from seg_type."""
         if source == "memory":
-            self.target_seg_type = "OBJ"
+            self.op_type = "WRITE"
+            self.initiator_seg_type = "DRAM"
         elif destination == "memory":
-            self.initiator_seg_type = "OBJ"
+            self.op_type = "READ"
+            self.initiator_seg_type = "DRAM"
         else:
-            raise ValueError(f"Invalid source for OBJ: {source}")
+            raise ValueError(f"Invalid source/destination for OBJ: source={source}, destination={destination}")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)